common/src/main/java/apoc/util/Util.java

```diff
@@ -670,8 +672,19 @@ public static String quote(String var) {
         return SourceVersion.isIdentifier(var) && !var.contains("$") ? var : '`' + var + '`';
     }
 
-    public static String sanitizeAndQuote(String var) {
-        return quote(var.replaceAll("`", ""));
+    // Escape all backticks in a string to stop Cypher Injection Attacks
+    public static String sanitizeBackTicks(String text) {
+        Pattern finOddCountBackticks = Pattern.compile("(?<!`)`(?:`{2})*(?!`)");
+        Matcher matcher = finOddCountBackticks.matcher(text);
```

Comment:
https://media2.giphy.com/media/13Qd06UWpuicJW/giphy.gif

Reply:
Looks good but could we add some tests for this to make sure we covered all cases and to prevent regressions in the future?